changelogs/fragments/3093-ejabberd_user-refactor.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - ejabberd_user - Replaced in-code check with ``required_if``, using ``get_bin_path()`` for the command, passing args to ``run_command()`` as list instead of string (https://github.com/ansible-collections/community.general/pull/3093).
```

Comment:
```suggestion
  - ejabberd_user - replaced in-code check with ``required_if``, using ``get_bin_path()`` for the command, passing args to ``run_command()`` as list instead of string (https://github.com/ansible-collections/community.general/pull/3093).
```